src/pages/public_registration/action_accordions/RefreshPublishingRequestButton.tsx

```diff
@@ -0,0 +1,24 @@
+import RefreshIcon from '@mui/icons-material/Refresh';
+import { LoadingButton } from '@mui/lab';
+import { t } from 'i18next';
```

Comment:
Bruk `useTranslation()`-hooken i stedet, da den tilpasser språket etter det som er valgt, mens når du importerer `t` direkte vil den alltid vise for språket som var valgt når den ble først lastet

```suggestion
import { useTranslation } from 'react-i18next';
```